.travis.yml

```diff
@@ -4,6 +4,9 @@
 language: objective-c
 
 env:
+  matrix:
+    
+    - CONDA_PERL=5.20.3.1
```

Comment:
I am not sure I like this. If `perl` is used in `build/run` I'd rather see them there. Like we have `numpy x.x` for example.